NOTES.md

```diff
@@ -1,3 +1,191 @@
+  * Lower the CPU load of an idle server (#3185)
+  * Now ignore the OS disk cache when calculating available memory (#3193)
+  * Now use `kqueue` instead of `poll` for events on OS X (#3403)
+  * Tables now become available faster and table metadata takes less space on disk (#3463)
+  * Queries that perform multiple HTTP requests now share the same cookies (#3469)
+  * Made resolving of ties in secondary indexes more consistent (#3550)
+  * New version notifications are added to the server logs (#3170)
+* Testing
+  * The ReQL tests can now run in parallel (#2305, #2672)
+  * The Ruby driver is now tested against different versions of Ruby (#2526)
+  * Relaxing floating point comparison to account for rounding issues (#2912)
+  * No longer depend on external HTTP servers (#3047)
+  * Switched to using the new cluster management features in the test suite (#3398, #3401)
+* ReQL
+  * Allow creating evenly-spaced shards (#1679)
+  * Allow querying the version of the server (#2698)
+  * Changefeeds are no longer aborted when the primary replica changes (#2757)
+  * Improved the error message when pseudo-types are used as objects (#2766)
+  * The names of types returned by `typeOf` and `info` now match (#3059)
+  * No longer silently ignore global optional arguments (#2052)
+  * Improved handling of socket, protocol and cursor errors (#3207)
+  * Added an `isOpen` method to connections (#2113)
+  * `info` on tables now works when `use_outdated` is true (#3355)
+  * `info` on tables now includes the table id (#3358)
+* Driver protocol
+  * Made internal names of commands more consistent (#3147)
+  * Added the `SUCCESS_ATOM_FEED` return type, used when returning changes for a single document (#3389)
+* JavaScript driver
+  * Upgrade to bluebird 2 (#2973)
+* Python driver
+  * Added a `next` method for cursors (#3042)
+  * `r.expr` now accepts any iterable or mapping (#3146)
+  * `rethinkdb export` now avoids travesing each table twice by using an estimated document count (#3483)
+* Build
+  * Updated code to build with post-Yosemite versions of XCode (#3219)
+* Packaging
+  * Added `procps` as a dependency in the Debian packages (#3404)
+  * Now depend on `jemalloc` instead of `gperftools` (#3419)
+
+## Bug Fixes ##
+
+* Tests
+  * Fixed the `TimerTest` test (#549)
+  * Always use the in-tree driver for testing (#711)
+* Server
+  * Consolidate log message into a single log entry (#925)
+  * Fixed continuation of partially-interrupted vectored I/O (#2665)
+* ReQL
+  * Changed type of between queries to from `TABLE` to the more correct `TABLE_SLICE` (#1728)
+* Web UI
+  * Use a real fixed width font for backtraces to make them align correctly (#2065)
+  * No longer show empty batches when more data is available (#3432)
+* Python driver
+  * Correctly handle unicode characters in error messages (#3051)
+* JavaScript driver
+  * Added support for the `batch_conf` argument to `run` (#3161)
```

Comment:
`batch_conf` suggests that that is the actual argument, but there is not such argument (instead there are a couple of them, such as `maxBatchRows` etc).
Maybe change this to something along the lines of "Added support for passing batch configuration arguments to `run` (#3161)"